pandas-stubs/core/indexes/base.pyi

```diff
@@ -470,7 +472,7 @@ class Index(IndexOpsMixin[S1], ElementOpsMixin[S1]):
         sort: bool = False,
     ) -> Index: ...
     @property
-    def values(self) -> np_1darray: ...
+    def values(self) -> np_1darray[Any]: ...
```

Comment:
This might be `np_1darray[S1]`, but that might create issues, because there are values of `S1` that wouldn't show up in a numpy array.  Might be worth trying `np_1darray[S1]` here and see what happens